Tests/MountebankSwiftTests/Api/MountebankIntegrationTests.swift

```diff
@@ -36,13 +36,42 @@ final class MountebankIntegrationTests: XCTestCase {
     }
 
     func testPostImposter() async throws {
-        let imposterResult = try await sut.postImposter(imposter: Imposter.Examples.includingAllStubs.value)
+        let imposterToPost = Imposter.Examples.includingAllStubs.value
+        let imposterResult = try await sut.postImposter(imposter: imposterToPost)
         guard imposterResult.port != nil else {
             XCTFail("Port should have been set by now.")
             return
         }
 
-        XCTAssertEqual(imposterResult, Imposter.Examples.includingAllStubs.value)
+        let result = Imposter(
+            port: imposterToPost.port,
+            networkProtocol: imposterToPost.networkProtocol,
+            name: imposterToPost.name,
+            stubs: imposterToPost.stubs,
+            defaultResponse: imposterToPost.defaultResponse,
+            recordRequests: imposterToPost.recordRequests,
+            numberOfRequests: 0,
+            requests: []
+        )
+
+        XCTAssertEqual(imposterResult, result)
+    }
+
+    func testGetImposter() async throws {
+        let port = try await postDefaultImposter(imposter: Imposter.Examples.simpleRecordRequests.value)
+        let httpClient = HttpClient()
+
+        let path = "/text-200"
+        let request = HTTPRequest(url: sut.makeImposterUrl(port: port).appending(path: path), method: .get)
+
+        _ = try await httpClient.httpRequest(request)
+        _ = try await httpClient.httpRequest(request)
+
+        let imposter = try await sut.getImposter(port: port)
+
+        XCTAssertEqual(imposter.requests?.count, 2)
+        XCTAssertEqual(imposter.requests?.first?.path, path)
+        XCTAssertEqual(imposter.requests?.first?.method, .get)
```

Comment:
Perhaps just assert agains a `Imposter.RecordedRequest(...)` object here to just check all properties? (Or an example file)

Reply:
This is not possible due to env that does impact the `RecordedRequest`. I've tested more properties + added a comment why not test against `RecordedRequest`